validator/rpc/server.go

```diff
@@ -76,8 +110,21 @@ func (s *Server) Start() {
 	}
 	s.grpcServer = grpc.NewServer(opts...)
 
+	// We create a new, random JWT key upon validator startup.
```

Comment:
JWT key has no relation to the hashed password stored in the disk right?

Reply:
Thats right. If this key changes, the worse that will happen is the user will have to login again